### PR TITLE
add login route and responses

### PR DESCRIPTION
**Pull request by mgjeffries**
When the client posts an email and password to the login path, the server will check to see if the user exists in the database and return a login token if the user exists.

To test, check out the matching branch on the client side, and verify that users can log in.

### CODE REVIEW

File: login/login.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    FROM user u
+    WHERE u.email = ?    
+    """, (loginCredentials['email'], ))    
+    data = db_cursor.fetchone()
+    
+    userMatches = db_cursor.rowcount
+    response = {}
+
+
+    if data != None:
```
Comment (skratz17) [inline login/login.py]: Do we need to add a password check here (or alternatively, in the query itself)? As of now it seems that we're only checking if there is a user with the given email, do we wanna verify their password now too?

Comment (mgjeffries) [reply login/login.py]: Good point. I just pushed up changes to check the password.

**Review by gavinschriver — APPROVED** [review]
workin here

**Comment by skratz17** [comment]
Oh yes

**Comment by skratz17** [comment]
oh oops that wasn't an approval that was just a comment one moment please

**Review by skratz17 — APPROVED** [review]

**Pull request merged**